modules/80_optimization/nash/postsolve.gms

```diff
@@ -323,26 +318,25 @@ if(p80_DevPriceAnticipGlobAllMax("2100") gt 0.1 * p80_surplusMaxTolerance("good"
 );
 
 ***additional criterion: did taxes converge? (only checked if cm_TaxConvCheck is 1)
-p80_taxrev_dev(t,regi) = 0;
-if (cm_TaxConvCheck eq 1,
-  loop(regi,
+p80_convNashTaxrev_iter(iteration,t,regi) = 0;
+loop(regi,
     loop(t,
-      if( abs(vm_taxrev.l(t,regi)) / vm_cesIO.l(t,regi,"inco") gt 1E-4,
-        p80_taxrev_dev(t,regi) = abs(vm_taxrev.l(t,regi)) / vm_cesIO.l(t,regi,"inco");
-        if (t.val lt 2100,
-          s80_bool = 0;
-          p80_messageShow("taxconv") = YES;
-        );
-      );
+         p80_convNashTaxrev_iter(iteration,t,regi) = abs(vm_taxrev.l(t,regi)) / vm_cesIO.l(t,regi,"inco");
```

Comment:
I would save this parameter without the `abs` transformation to save the sign information for further debugging if necessary.

Reply:
also before we did only use the abs info here

Reply:
but it would be better to save the info without the absolute value, like we do for other tests, and only use the absolute transformation in the conditional checks.

Reply:
but here the abs() only holds for `vm_taxrev` and not the whole term

Reply:
`vm_cesIO` is a quantity and declared as a positive variable, so it is always positive. Therefore, applying the abs() to the numerator or to both the numerator and denominator makes no difference.